pkg/synthetictests/duplicated_events.go

```diff
@@ -185,6 +185,11 @@ var knownEventsBugs = []knownProblem{
 		BZ:       "https://bugzilla.redhat.com/show_bug.cgi?id=2017435",
 		Topology: topologyPointer(v1.SingleReplicaTopologyMode),
 	},
+	// builds tests trigger many changes in the config which creates new rollouts -> event for each pod
+	{
+		Regexp:    regexp.MustCompile(`ns/openshift-route-controller-manager deployment/route-controller-manager - reason/ScalingReplicaSet \(combined from similar events\): Scaled down replica set route-controller-manager-[a-z0-9-]+ to [0-9]+`),
+		TestSuite: stringPointer("openshift/build"),
```

Comment:
I have kept this in the `knownEventsBugs` section since it is the only one that distignuishes by `TestSuite` even though it is not a bug.

Should I do a refactoring and move it somewhere else?

Reply:
i think this is fine.  You might make it clearer in the comment that this is not considered a bug, but rather "working as deisgned and needs to be tolerated" though.

Are the tests in question *only* part of the `openshift/build` suite?

Reply:
@bparees good suggestion - updated

we are observing these only in builds suite atm, but it could occur to anybody who does a lot of changes to either of

- `builds.config.openshift.io/cluster`
- `images.config.openshift.io/cluster`
- `configmap/openshift-controller-manager-images` in `openshift-controller-manager-operator` ns
- `openshiftcontrollermanagers.operator.openshift.io/cluster`